src/util.ts

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+import * as fs from 'fs';
+import * as request from 'request';
+import * as rp from 'request-promise';
+import * as chalk from 'chalk';
+import * as appInsights from 'applicationinsights';
+
+let insight = appInsights.getClient('78cc7757-c7a2-4382-b801-bce73cf33d7a');
```

Comment:
What's the usual way of doing this? Should I just export it from here? Or stick it in package.json and grab it somehow? Or make a new file?